typescript/db/src/resolvers/workspace.ts

```diff
@@ -153,10 +153,15 @@ const memberships = async (parent: DbWorkspaceWithType) => {
 
 const datasets = async (parent: DbWorkspaceWithType) => {
   const db = await getPrismaClient();
-  return await db.dataset.findMany({
+  const queryResult = await db.dataset.findMany({
     where: { workspaceSlug: parent.slug },
     orderBy: { createdAt: Prisma.SortOrder.asc },
   });
+
+  return queryResult.map((datasetWithoutTypename) => ({
+    ...datasetWithoutTypename,
+    __typename: "Dataset",
```

Comment:
On the long-term, we'll really need to find a way to get rid of these things 😢

In the meantime, can you use an hypothetical `addTypenames` function which would loop over an array of objects to call another hypothetical `addTypename` function?